packages/kbn-monaco/src/painless/lexer_rules/painless.ts

```diff
@@ -41,8 +34,7 @@ interface Language extends monaco.languages.IMonarchLanguage {
 }
 
 export const lexerRules = {
```

Comment:
As you can see, there are a few minor differences between the painless lexer rules in xjson ([original code](https://github.com/elastic/kibana/blob/master/packages/kbn-monaco/src/xjson/lexer_rules/painless.ts)) and the lexer rules in painless lab ([original code](https://github.com/elastic/kibana/blob/master/x-pack/plugins/painless_lab/public/lib/monaco_painless_lang.ts)). I don't see why they would/should be different, but if there is a reason, please let me know.